Dockerfile

```diff
@@ -28,7 +28,10 @@ RUN apt-get update &&\
                        graphviz \
                        libhdf5-dev \
                        swig &&\
-    ln -s /usr/bin/python3 /usr/local/bin/python &&\
+    add-apt-repository -y ppa:deadsnakes/ppa &&\
+    apt install -y python3.7 \
+                   python3.7-dev &&\
+    ln -s /usr/bin/python3.7 /usr/local/bin/python &&\
```

Comment:
Should look into install pip linked to this Python rather than install regular pip3

Reply:
I made it so `pip` is linked to `python3.7`. I also tried getting rid of `python3-pip` but then it complains about `pip` not being a module when running a [command](https://github.com/cblessing24/pytorch-docker/blob/8e195c8b717c1cd0e9b6cc573f713d629ff2305b/Dockerfile#L34).